package.json

```diff
@@ -83,21 +83,17 @@
     "lint:scss": "stylelint src/**/*.scss",
     "precompile": "rm -rf lib",
     "prepare": "npm run compile",
-    "start": "tt-serve",
-    "start-prod": "tt-serve -p",
-    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
-    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
+    "prepare-for-release": "terra prepare-for-release",
+    "start": "webpack-dev-server",
+    "start-prod": "webpack-dev-server -p",
```

Comment:
shouldn't this be terra express-server for testing prod?

Reply:
Start-static, down below is testing the static site in prod mode.